src/modules/formBuilder/components/itemEditor/conditionals/InitialValue.tsx

```diff
@@ -84,12 +84,20 @@ const InitialValue: React.FC<Props> = ({ itemType, control }) => {
                 control={control}
                 defaultValue={InitialBehavior.IfEmpty}
                 shouldUnregister
-                render={({ field: { ref, value, onChange, ...field } }) => (
+                rules={{ required: true }}
+                render={({
+                  field: { ref, value, onChange, ...field },
+                  fieldState: { error },
+                }) => (
```

Comment:
Could DRY up the RadioGroupInput in `ManageEnableWhen` by refactoring out to a ControlledRadioGroupInput like we have for some others. There are some differences in implementation-- like the other sets `helperText={error?.message}` which is more flexible for supporting any kind of error.

Reply:
Good idea to dry up into a controlled radio component, I implemented that.

I was seeing that the `error` object coming from RHF did not actually have a message on it, just a type of 'required.' I can investigate a bit more to double check that behavior and see if I can find where it's coming from.

Reply:
Ah it was a pretty simple fix. https://github.com/greenriver/hmis-frontend/pull/885/commits/26082bd076389dcaef16a4d83246dc7e86fc1a64